config/locales/en.yml

```diff
@@ -9,6 +9,9 @@ en:
         consent_modal:
           description: This modal gives the opportunity to filter the cookies you
             want.
+          privacy_policy:
+            name: privacy policy
+            text: To learn more, please read our {privacyPolicy}.
```

Comment:
I'm not sure about the variable interpolation, can you check it's not 
```suggestion
            text: To learn more, please read our %{privacyPolicy}.
```

Reply:
I checked with the v.0.21 (deployed on CultuurConnect) and it seems its `{privacyPolicy}`

Reply:
You're right, this is a js interpolation.